makefile.shade

```diff
@@ -151,27 +165,21 @@ var buildTarget = "compile"
                 }
             });
         }
+        
+        var commitsAsString = string.Join(Environment.NewLine, commits.Select(c => c.Key + ":" + c.Value));
+        File.WriteAllText(Path.Combine(universeArtifacts, "commits"), commitsAsString);
     }
-
-#smoke-test-mono .pull .fix-project-json .change-default-build-target-to-verify .build-all
-
-#fix-project-json
+    
+#remove-src-folders
     @{
-        repositories = repositories.Except(excludeReposOnMono).ToArray();
+        foreach (var repo in GetRepositoriesToBuild())
```

Comment:
Spoke to Andrew and we don't actually need the project reference there. I'll send a PR to change it. Do you know of other repos that use `target: project`